photon-api/src/main/scala/com/linkedin/photon/ml/supervised/regression/LinearRegressionModel.scala

```diff
@@ -30,30 +31,69 @@ class LinearRegressionModel(override val coefficients: Coefficients)
   with Regression
   with Serializable {
 
+  /**
+   *
+   * @return The model type
+   */
+  override def modelType: TaskType = LINEAR_REGRESSION
+
+  /**
+   * Compute the mean of the linear regression model.
```

Comment:
The model is also a mean in a Bayesian sense, but that's not what this function is for. So it should be response mean.